app/tests/utils/dictCmpTests.py

```diff
@@ -18,9 +18,85 @@ def test_two_flat_dicts_equal(self):
             "project": "lepv"
         }
 
+        dict_3 = {}
+
+        dict_4 = {}
+
+        comp_result_1 = DictUtil.compare(dict_1, dict_2)
+        comp_result_2 = DictUtil.compare(dict_3, dict_4)
```

Comment:
we should have separate test case for this, say "two_empty_dicts_should_equal".

Reply:
ok